Add section: About This Handbook, under construction (#31)
hyphacoop/handbook: README.md

Hypha Organizational Handbook (H₂O) describes the vision, processes, and culture of Hypha Worker Co-operative. 🌿🍄

### About This Handbook
<!-- Derived from: https://handbook.enspiral.com/#about-this-handbook -->

> This Handbook's primary audience is Hypha worker-members and their

Reviewer: ```suggestion
This Handbook's primary audience is Hypha member-owners and their
```

Reviewer: We've approved language which calls ourselves "member-owners" at a previous meeitng

Author: I'm easy, but tried to search our notes, and only found this, where the notes didn't reflect that _per se_: https://github.com/hyphacoop/organizing/blob/master/2019-09-04-all-hands-meeting.md

Something else you were thinking? Or were notes missing?

Reviewer: Pretty sure we discussed this when we finalized membership classes (https://github.com/hyphacoop/organizing/blob/master/2019-05-27-all-hands-meeting.md), but I don't see it in notes. We updated the business plan to "member-owner" afterwards and issues use that language too....

Author: don't recall deciding such a long time ago, thought it was much more recent, but glad to have one or the other decided :) let's get this in glossary in near future!

> outside collaborators, but it will be publicly available for others who

Reviewer: ```suggestion
collaborators, but it will be publicly available for others who
```

Author: okie doke. added here: https://github.com/hyphacoop/handbook/issues/4#issuecomment-556360041

> might find it useful. The goal is to give all of us as much clarity and

Reviewer: ```suggestion
might find it useful. The goal is to provide as much clarity and
```

context as possible, while sharing our structures and practices with the
outside world as well.

![Old-timey web 1.0 UNDER CONSTRUCTION banner](images/under-construction.gif)<br />

> This resource is in perpetual beta, and is constantly changing. If you

Reviewer: ```suggestion
This resource is in perpetual beta and constantly changing. If you
```

> see something that could be improved: that's your invitation to improve

Reviewer: ```suggestion
see something that could be improved, consider this your invitation to improve
```
